app/javascript/components/venue/VenueEdit.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { Component } from 'react';
+import Axios from 'axios';
+import Grid from '@material-ui/core/Grid';
+import TextField from '@material-ui/core/TextField';
+
+export default class VenueEdit extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      venue: { address: '', description: '', people_capacity: '' }
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange = venue => {
+    const params = this.state;
+    params['venue'][venue.target.name] = venue.target.value;
+    this.setState({
+      params
+    });
```

Comment:
Check at how handlechange is written in Events. 
You write twice in state

Reply:
rewritten